packages/fxa-settings/src/pages/PostVerify/ThirdPartyAuthCallback/index.tsx

```diff
@@ -3,40 +3,150 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React, { useEffect } from 'react';
-import { FtlMsg } from 'fxa-react/lib/utils';
+import {
+  FtlMsg,
+  hardNavigate,
+  hardNavigateToContentServer,
+} from 'fxa-react/lib/utils';
 import { RouteComponentProps, useNavigate } from '@reach/router';
 import LoadingSpinner from 'fxa-react/components/LoadingSpinner';
 import AppLayout from '../../../components/AppLayout';
-import Storage from '../../../lib/storage';
+import { AUTH_PROVIDER } from 'fxa-auth-client/browser';
+import { useAccount } from '../../../models';
+import {
+  StoredAccountData,
+  persistAccount,
+  setCurrentAccount,
+} from '../../../lib/storage-utils';
+import { sessionToken } from '../../../lib/cache';
+
+// TODO this page to be completed/activated in FXA-8834
```

Comment:
Just a drive-by comment: when we do this, I think we're going to want to rework some of this to use `useValidatedQueryParams`.

Reply:
Added a TODO comment for FXA-8834 (and updated the description there too!)